run/O2PrimaryServerDevice.h

```diff
@@ -60,6 +62,31 @@ class O2PrimaryServerDevice final : public FairMQDevice
     TStopwatch timer;
     timer.Start();
     auto& conf = o2::conf::SimConfig::Instance();
+
+    // init magnetic field as it might be needed by the generator
+    int fld = conf.getConfigData().mField, fldAbs = std::abs(fld);
+    float fldCoeff;
+    o2::field::MagFieldParam::BMap_t fldType;
+    switch (fldAbs) {
```

Comment:
ok for me. Just wondering if we use similar code elsewhere (and if we should put this into a dedicated function)?